packages/ceramic-doctype-tile/src/__tests__/three-id.test.ts

```diff
@@ -80,15 +80,15 @@ const RECORDS = {
           }
         ],
         "prev": "bafybeig6xv5nwphfmvcnektpnojts33jqcuam7bmye2pb54adnrtccjlsu",
-        "header": {}
+        "header": {
+          "nonce": 0
```

Comment:
Possible to avoid setting nonce to 0? It could be implicitly 0 right?

Reply:
yep, just wanted to have it here because of consistency.